ui/entry-perf.js

```diff
@@ -45,6 +45,7 @@ import './css/treeherder-global.css';
 import './css/treeherder-navbar.css';
 import './css/perf.css';
 import './css/treeherder-loading-overlay.css';
+import './css/treeherder-notifications.css';
```

Comment:
I think lets not import that and use notifications unless we're doing it in all of Perfherder.